wordle-web/pages/game.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <v-container fluid fill-height class="pa-0">
+    <v-btn outlined absolute top right color="primary" @click="dialog = true">
+      <v-icon primary> mdi-account </v-icon>
+      {{ playerName }}
+    </v-btn>
+
+    <valid-words :wordle-game="wordleGame" />
+    
+    <v-dialog v-model="dialog" width="450" persistent>
+      <v-card>
+        <v-card-title>Enter Your Name!</v-card-title>
+        <v-card-text>
+          Add you name to our score board so you can save your game scores!
+          <v-text-field v-model="playerName"></v-text-field>
+        </v-card-text>
+        <v-card-actions>
+          <v-btn @click="addPlayer"> Submit </v-btn>
+          <v-btn @click="dialog = false"> I prefer to remain nameless </v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+
+    <v-row>
+      <v-spacer />
+      <v-alert v-if="wordleGame.gameOver" :type="gameResult.type" width="450">
+        {{ gameResult.text }}
+        {{ setGameTime() }}
+        {{ addStats() }}
```

Comment:
I would suggest calling these in the submitWord function, rather than when the banner appears. It doesn't seem to save correctly when placed here?